carsus/io/kurucz/gfall.py

```diff
@@ -145,12 +150,23 @@ def read_gfall_raw(self, fname=None):
 
         field_type_dict = {col: dtype for col,
                            dtype in zip(self.gfall_columns, field_types)}
-        gfall = pd.read_fwf(fname, widths=field_widths, skip_blank_lines=True,
+
```

Comment:
Get MD5 from local or remote files (http). Returns the same checksum than `md5sum` in the command line!